client.go

```diff
@@ -439,3 +441,45 @@ func (c *Client) Webhook(header http.Header, body []byte) (*Webhook, error) {
 	}
 	return nil, errors.New("Invalid webhook")
 }
+
+/*
+Notify is used to send native push notifications via Apple APNS or Google GCM/FCM systems. Please make sure that
+you have provided a Client.PushNotificationHost, please see Pusher docs for details: https://pusher.com/docs/push_notifications
+*/
+func (c *Client) Notify(interests []string, pushNotification PushNotification) (*NotifyResponse, error) {
+	pNRequest := notificationRequest{
+		interests,
+		&pushNotification,
+	}
+
+	err := pNRequest.validate()
+	if err != nil {
+		return nil, err
+	}
+
+	if c.PushNotificationHost == "" {
+		return nil, errors.New("no Client.PushNotificationHost provided")
+	}
+
+	requestBody, err := json.Marshal(pNRequest)
+	if err != nil {
+		return nil, err
+	}
+
+	path := fmt.Sprintf("/%s/%s/apps/%s/notifications", PushNotifAPIPrefixDefault, PushNotifAPIVersionDefault, c.AppId)
+
+	url, err := createRequestURL("POST", c.PushNotificationHost, path, c.Key, c.Secret, authTimestamp(), c.Secure, requestBody, nil, c.Cluster)
```

Comment:
This sucks but using http.MethodPost actually breaks the librabry for go versions < 1.6, they added the const methods then!